docs/user/REST.rst

```diff
@@ -2,6 +2,261 @@
+            num_units=128,
+            num_layers=2,
+            dropout=0,
+            vocab_size=1000,
+        ):
+            super().__init__()
+            self.embedding_dim = embedding_dim
+            self.rec_layer_type = rec_layer_type.lower()
+            self.num_units = num_units
+            self.num_layers = num_layers
+            self.dropout = dropout
+
+            self.emb = nn.Embedding(
+                vocab_size + 1, embedding_dim=self.embedding_dim)
+
+            rec_layer = {'lstm': nn.LSTM, 'gru': nn.GRU}[self.rec_layer_type]
+            # We have to make sure that the recurrent layer is batch_first,
+            # since sklearn assumes the batch dimension to be the first
+            self.rec = rec_layer(
+                self.embedding_dim, self.num_units,
+                num_layers=num_layers, batch_first=True,
+                )
+
+            self.output = nn.Linear(self.num_units, 2)
+
+        def forward(self, X):
+            embeddings = self.emb(X)
+            # from the recurrent layer, only take the activities from the
+            # last sequence step
+            if self.rec_layer_type == 'gru':
+                _, rec_out = self.rec(embeddings)
+            else:
+                _, (rec_out, _) = self.rec(embeddings)
+            rec_out = rec_out[-1]  # take output of last RNN layer
+            drop = F.dropout(rec_out, p=self.dropout)
+            # Remember that the final non-linearity should be softmax, so
+            # that our predict_proba method outputs actual probabilities!
+            out = F.softmax(self.output(drop), dim=-1)
+            return out
+
+
+You can find the full contents of the ``model.py`` file in the
+``skorch/examples/rnn_classifer`` folder of Skorch's source code.
+
+Now with dataset and model in place, it's time to try Palladium out.
+You can install Palladium with ``pip install palladium``.
+
+From within the directory that contains ``model.py`` and
+``palladium-config.py`` now run the following command::
+
+  PALLADIUM_CONFIG=palladium-config.py pld-fit --evaluate
+
+You should see output similar to this::
+
+  INFO:palladium:Loading data...
+  INFO:palladium:Loading data done in 0.607 sec.
+  INFO:palladium:Fitting model...
+    epoch    train_loss    valid_acc    valid_loss     dur
+  -------  ------------  -----------  ------------  ------
+        1        0.7679       0.5008        0.7617  3.1300
+        2        0.6385       0.7100        0.5840  3.1247
+        3        0.5430       0.7438        0.5518  3.1317
+        4        0.4736       0.7480        0.5424  3.1373
+        5        0.4253       0.7448        0.5832  3.1433
+  INFO:palladium:Fitting model done in 29.060 sec.
+  DEBUG:palladium:Evaluating model on train set...
+  INFO:palladium:Train score: 0.83068
+  DEBUG:palladium:Evaluating model on train set done in 6.743 sec.
+  DEBUG:palladium:Evaluating model on test set...
+  INFO:palladium:Test score:  0.75428
```

Comment:
Maybe a short comment on where this score comes from (referring back to the test dataset).